package.json

```diff
@@ -62,7 +63,8 @@
     "react-hooks-testing-library": "^0.5.0",
     "react-test-renderer": "^16.8.6",
     "rimraf": "^2.6.3",
-    "standard-version": "^6.0.1"
+    "standard-version": "^6.0.1",
+    "typescript": "^3.4.5"
```

Comment:
are you sure this is necessary? I have looked at another couple repos exporting typings and can't see them referencing typescript in their deps

Reply:
Ahh, I see. Lemme double check

Reply:
You we're right. Good catch!

Reply:
great thanks. last thing, because the package no longer needs to be included, then I guess no update to the package-lock file should be necessary

Reply:
Duh. Silly me. Sorry about that

Reply:
no worries. thanks a lot for the contribution. gonna merge this